dropwizard-cassandra/build.gradle

```diff
@@ -1,9 +1,9 @@
 dependencies {
 	compile project(':dropwizard-guice')
 
-	compile "smartthings:cassandra-migration:0.0.11"
-	compile "com.datastax.cassandra:cassandra-driver-core:2.1.10.3"
-	compile "com.datastax.cassandra:cassandra-driver-mapping:2.1.10.3"
+	compile "smartthings:cassandra-migration:0.1.0"
```

Comment:
`cassandra-migration:0.0.11` depends on some internal schema of cassandra 2.1 so moving to `cassandra-migration:0.1.0` but that depends on `cassandra-core-driver:3.1.0` so we are updating to 3.1.0